tailwind.config.mjs

```diff
@@ -199,8 +199,12 @@ export default {
       // class to the sidebar frame when it's collapsed. This modifier allows
       // sub-components to select for that state.
       addVariant('sidebar-collapsed', '.sidebar-collapsed &');
+      // Custom variant for a specific use case in which an element needs to
+      // style itself based on the hover states of TWO parents (.group and
+      // .hover-group). See `src/annotator/components/AdderToolbar`
+      addVariant('hover-group-hover', '.group:hover .hover-group:hover &');
```

Comment:
I'm not gonna mince words: this is a tad fugly. I hope the comments everywhere help. I've just never quite run into a situation like this before, where the hover state of two ancestors has to be assessed.